src/main/java/au/net/immortius/wardrobe/site/entities/SameSkinEntry.java

```diff
@@ -0,0 +1,19 @@
+package au.net.immortius.wardrobe.site.entities;
+
+public class SameSkinEntry {
```

Comment:
Doesn't look like this is used anywhere?

Reply:
Jup this seems to be dead code from a previous implementation. I'll remove it